docs/docs/references/dbt-write-back.mdx

```diff
@@ -0,0 +1,56 @@
+import dbtWriteBack from './assets/dbt-write-back.jpg';
+import WriteBackPullRequest from './assets/write-back-pull-request.jpg';
+import TablesConfigurationWriteBack from './assets/tables-configuration-write-back.jpg';
+
+# dbt Write-Back
```

Comment:
content looks good - like i mentioned elsewhere i think this should just be a mention in the SQL runner docs with a link to a new section in the dbt Projects integrations reference doc